benchmarks/openh264_decoder_fuzzer/build.sh

```diff
@@ -20,7 +20,7 @@ svn export https://github.com/mozillasecurity/fuzzdata.git/trunk/samples/h264 co
 mv ./res/*.264 ./corpus/
 zip -j0r ${OUT}/decoder_fuzzer_seed_corpus.zip ./corpus/
 
-# build 
+# build
```

Comment:
Try not to do unrelated changes in a PR like this. It's hard enough to manage as is.